### PR TITLE
Added import code review agent skill

### PR DESCRIPTION
**Pull request by rohitkumarbhagat**

### CODE REVIEW

File: .agents/skills.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
     {
       "path": "agents/skills/dc-import-diagnostics"
     },
+    {
+      "path": "agents/skills/dc-import-code-review"
+    },
```
[the remark this answers is not in the document]
Comment (rohitkumarbhagat) [reply .agents/skills.json:8]: that is intentional. Its not a intended to be used by everyone, else avoided auto loading by not adding to skills.json

File: agents/skills/dc-import-review-signal-miner/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+remove an unverified path or the active repository.
+
+Verify that the snapshot resolves to the fetched `master` commit. A worktree
+provides isolation; the shallow, partial, and sparse options reduce downloaded
+data. Remove only temporary paths created by this run after both reports have
+been written successfully.
+
+## Collect pull requests and comments
+
+Use GitHub REST API version `2026-03-10`. Do not silently fall back to an
```
[the remark this answers is not in the document]
Comment (rohitkumarbhagat) [reply agents/skills/dc-import-review-signal-miner/SKILL.md:94]: Thanks. I verified this against GitHub’s current API-version documentation and the live /versions endpoint. 2026-03-10 is supported, and a request with that header returns HTTP 200 with X-GitHub-Api-Version-Selected: 2026-03-10. 2022-11-28 is also supported but is not the only supported version, so I’m retaining 2026-03-10.

File: agents/common/scripts/skill_contract_test.py [hunk 3]
```diff
@@ -428,5 +428,76 @@ def test_runtime_environment_registry_is_minimal_and_complete(self):
...
+        for source in sources:
+            text = source.read_text(encoding='utf-8')
+            with self.subTest(source=source.name):
+                self.assertNotIn('/Users/', text)
+                self.assertNotIn('file://', text)
+                self.assertNotIn('<REPO_ROOT>/', text)
+
+
 if __name__ == '__main__':
     unittest.main()
```
[the remark this answers is not in the document]
Comment (rohitkumarbhagat) [reply agents/common/scripts/skill_contract_test.py:499]: Thanks for the suggestion. The signal miner is intentionally not registered because it is a specialized, manually invoked skill. Instead of adding skill-specific tests that duplicate structural or prose assertions, I updated the generic Markdown referential-integrity test to discover every agents/skills/*/SKILL.md, including unregistered skills. This ensures the miner and all its linked guidance remain valid without coupling the test to its wording or registration.

**Review by ajaits — APPROVED** [review]
